[2024-08-21 19:05:50,746] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: captors_dag.task_3 2024-08-21T19:05:47.227767+00:00 [queued]>
[2024-08-21 19:05:50,761] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: captors_dag.task_3 2024-08-21T19:05:47.227767+00:00 [queued]>
[2024-08-21 19:05:50,762] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2024-08-21 19:05:50,762] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2024-08-21 19:05:50,763] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2024-08-21 19:05:50,774] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): task_3> on 2024-08-21T19:05:47.227767+00:00
[2024-08-21 19:05:50,778] {standard_task_runner.py:52} INFO - Started process 49 to run task
[2024-08-21 19:05:50,782] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'captors_dag', 'task_3', '2024-08-21T19:05:47.227767+00:00', '--job-id', '5', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/mod_captors.py', '--cfg-path', '/tmp/tmp361x2c2s', '--error-file', '/tmp/tmpudf7qg49']
[2024-08-21 19:05:50,783] {standard_task_runner.py:77} INFO - Job 5: Subtask task_3
[2024-08-21 19:05:50,827] {logging_mixin.py:104} INFO - Running <TaskInstance: captors_dag.task_3 2024-08-21T19:05:47.227767+00:00 [running]> on host ad333fe07a6a
[2024-08-21 19:05:50,870] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=captors_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2024-08-21T19:05:47.227767+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-21T19:05:47.227767+00:00
[2024-08-21 19:05:50,883] {python.py:118} INFO - Done. Returned value was: None
[2024-08-21 19:05:50,900] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=captors_dag, task_id=task_3, execution_date=20240821T190547, start_date=20240821T190550, end_date=20240821T190550
[2024-08-21 19:05:50,933] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-21 19:05:50,957] {local_task_job.py:146} INFO - Task exited with return code 0
